Sources/StreamVideoSwiftUI/Models/IncomingCall.swift

```diff
@@ -17,18 +17,21 @@ public struct IncomingCall: Identifiable, Sendable, Equatable {
     public let type: String
     public let members: [Member]
     public let timeout: TimeInterval
+    public let video: Bool
 
     public init(
         id: String,
         caller: User,
         type: String,
         members: [Member],
-        timeout: TimeInterval
+        timeout: TimeInterval,
+        video: Bool
```

Comment:
Should we add a default value (to false) here, since it's a public init?

Reply:
The other values don't have though and it will make the parameter look "less" important. Even if people are using it it will be a very small number of them affected. Up to you.